ONSdigital/ras-rm-auth-service
# Update dependencies

# Motivation and Context
Fixes the security issue.  Additionally this fixes the linting issue found by flake8 and one of the deprecation warnings that the unit tests flagged up

# What has changed




# How to test?
Create and change a user.  No functionality should be different.

If using docker for all the services and running this service locally, you'll need to make this change to the.env file to make it look at your local copy:
```bash
-OAUTH_HOST=auth-service
+OAUTH_HOST=docker.for.mac.localhost
```
You also need to export 
```bash
export SECURITY_USER_NAME='admin'
export SECURITY_USER_PASSWORD='secret'
```
or just modify the config to default the security user name and password

# Links
https://trello.com/c/Fm3pOacm/756-spike-migrate-django-to-ras-rm-auth-service-4-days
